src/hyperfine/internal.rs

```diff
@@ -97,6 +103,39 @@ pub fn min(vals: &[f64]) -> f64 {
         .unwrap()
 }
 
+pub fn write_benchmark_comparison(results: &Vec<ExportEntry>) {
+    if results.len() < 2 {
+        return;
+    }
+
+    // Show which was faster, maybe expand to table later?
+    let mut fastest_item: &ExportEntry = &results[0];
+    let mut longer_items: Vec<&ExportEntry> = Vec::new();
+
+    for run in &results[1..] {
```

Comment:
It looks like we could use [sort_by_key](https://doc.rust-lang.org/stable/std/primitive.slice.html#method.sort_by_key) here, instead of this loop. Something like:
``` rust
let mut results_sorted = results.clone();
results_sorted.sort_by_key(|r| r.mean);
```

Reply:
Oh, maybe we need to use `sort_by` like you did below.

Reply:
Yeah, partitioning/sorting these elements was a bit tricky